<launch>
   <node name="lx_camera_node" pkg="lx_camera_ros" type="lx_camera_node" output="screen">

	<!-- ip、日志路径、流配置、算法、工作模式配置、点云单位 -->
	<param name="ip" 							value="0"/>
	<param name="log_path" 						value="/var/log/"/>
	<param name="is_xyz" 						value="1" />
	<param name="is_depth" 						value="1" />
	<param name="is_amp" 						value="1" />
	<param name="is_rgb" 						value="1" />
	<param name="lx_work_mode" 					value="0" />
	<param name="lx_application" 				value="1" />
	<param name="lx_tof_unit" 					value="1" />

	<!-- 相机位姿配置 -->
	<param name="x" 							value="0" />
	<param name="y" 							value="0" />
	<param name="z" 							value="0" />
	<param name="yaw" 							value="0" />
	<param name="roll" 							value="0" />
	<param name="pitch" 						value="90" />
	
	<!-- 是否使用launch配置 -->
	<param name="raw_param" 						value="0" />

	<!-- 2D配置 -->
	<param name="lx_2d_binning" 				value="0" />
	<param name="lx_2d_undistort" 				value="0" />
	<param name="lx_2d_undistort_scale" 		value="51" />
	<param name="lx_2d_auto_exposure" 			value="0" />
	<param name="lx_2d_auto_exposure_value" 	value="11" />
	<param name="lx_2d_exposure" 				value="10001" />
	<param name="lx_2d_gain" 					value="101" />

	<!-- 3D配置 -->
	<param name="lx_rgb_to_tof" 				value="0" />
	<param name="lx_3d_binning" 				value="0" />
	<param name="lx_mulit_mode" 				value="0" />
	<param name="lx_3d_undistort" 				value="0"/>
	<param name="lx_3d_undistort_scale" 		value="0" />
	<param name="lx_hdr" 						value="0" />
	<param name="lx_3d_auto_exposure" 			value="1" />
	<param name="lx_3d_auto_exposure_value" 	value="50" />
	<param name="lx_3d_first_exposure" 			value="1100" />
	<param name="lx_3d_second_exposure" 		value="200" />
	<param name="lx_3d_gain" 					value="11" />
	
	<!-- 深度 -->
	<param name="lx_min_depth" 					value="0" />
	<param name="lx_max_depth" 					value="8000" />

   </node>

</launch>
